### ISSUE TITLE
Skinning for "Show Desktop" button

### ISSUE THREAD

**Issue by bonzibudd** [Open-Shell/Open-Shell-Menu#603]
**Is your feature request related to a problem? Please describe.**
Open-Shell has a lot of capabilities in Taskbar skinning, but it is limited in it's ability to customize taskbar buttons. If you are using a specific styled theme, then the flat Win10 buttons can sometimes stand out compared to the rest of the theme.

**Describe the solution you'd like**
I would like for Open-Shell to be able to skin the Show Desktop button on the Taskbar using 3 states: inactive, hot, and pressed.

**Describe alternatives you've considered**
I know that StartIsBack++ and WindowBlinds have this ability, and I think it would be great if Open-Shell could do something similar. 

Here is the Windows 7 Show Desktop button for reference:
![image](https://user-images.githubusercontent.com/61938331/104107144-3d281900-5288-11eb-8f6d-b7c6e94d1508.png)

**Comment by TheEvilKingBowser** [comment]
Agreed! I'd absolutely love to be able to skin the "Show desktop" button in Windows 10 without StartIsBack or WindowBlinds.
